### ISSUE TITLE
Is it possible to add folder recognition instead of ozp files

### ISSUE THREAD

**Issue by lucienlmy** [iquercorb/OpenModMan#131]
![image](https://github.com/user-attachments/assets/b9665bf3-af14-4936-ad5c-788e73523e6a)

**Comment by iquercorb** [comment]
I don't understand your question. What "recognition" are you talking about ?

**Comment by lucienlmy** [comment]
> I don't understand your question. What "recognition" are you talking about ?

I mean detect folder type

**Comment by lucienlmy** [comment]
> I am not sure I well understand, but it seam to me that what you are asking for is already implemented... Simply put a folder into your Library folder, and you will see it as Mod, with folder icon... The only thing you cannot do, is to drag & drop directly into OMM, you need to do this from file explorer.

Folders can indeed be opened. I noticed that when the software is opened, the window does not remember the position when it was last closed. It seems to always be centered based on the left border. For example, if I stretch the right border of the window a little to the right, it will cover part of the right side when I close the software and open it again. If I maximize the window, close it and open it again, it will still cover part of the right side.

Original state：
![image](https://github.com/user-attachments/assets/764b7e25-a70a-465d-8a85-ae2686bc8528)

The window is a bit longer, and the right side will be covered when you open the software again：
![image](https://github.com/user-attachments/assets/d68047bc-5000-4128-b1a9-89a834fe72c6)

I maximized the window, closed it, and reopened it again：
![image](https://github.com/user-attachments/assets/d6f367b6-c686-47eb-b99a-fa1c865e0f18)

It always uses the left border as the reference. Can it be modified so that it always opens based on the window size and keeps the center as the reference? Or can it record the size and position on the screen before closing the window. The next time it is opened, it will automatically read the last window record.
